source/clock.rst

```diff
@@ -0,0 +1,143 @@
+==========
+Clock
+==========
+
+About
+-----
+
+The Clock activity is an education and teaching tool to help children learn how to read time. This gives them a way to know the time with their XO even if they do not own a watch.
```

Comment:
Write directly for the audience rather than for the teachers of the audience; so "an education and teaching tool to help children learn how to read time" could be something like "is for learning how to read time."

See what you can do to remove redundant words.  e.g. consider each word, or pair of words, or triplet, and see if you can replace them with nothing, or a smaller number of words.

Avoid the term "XO", as it is not necessary now that Sugar is hardware agnostic.